objects/endpoint.rb

```diff
@@ -79,7 +79,9 @@ def to_h
       log: @item['log']
     }
     h[:updated] = Time.at(@item['updated']) if @item['updated']
-    h[:flipped] = Time.at(@item['flipped']) if @item['flipped']
+    # TODO: #75 consider other implementations of the null Time object that may
+    # be better than Time.new(0).
+    h[:flipped] = @item['flipped'] ? Time.at(@item['flipped']) : Time.new(0)
```

Comment:
I think the proper way would need some small refactoring but did this to solve the bug and think about that later (that was the puzzle about).